### PR TITLE
Typography page

### PR DESCRIPTION
**Pull request by tobiasbu**
# What

After a storm in the galaxy I opening this pull request with Typography page 🔭 

# Why

Unfinished challenge from hackathon which desires to be finished.

# How

Writing a simple `.mdx` file.

### CODE REVIEW

File: packages/docs/src/components/Sidebar.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ const Backdrop = styled.div`
   }
 `;
 
-function Sidebar({...props}) {
+function Sidebar({ ...props }) {
```
Comment (ericaprieto) [inline packages/docs/src/components/Sidebar.js]: this could be just `function Sidebar(props) {`

File: packages/docs/src/pages/typography.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  scope={{ Text }}
+  code={
+  `<>
+  <Text fontSize="large">A B C D E F G H I J K L M N O P Q R S T U V W X Y Z</Text>
+  <Text fontSize="large">a b c d e f g h i j k l m n o p q r s t u v w x y z</Text>
+  <Text fontSize="large">0 1 2 3 4 5 6 7 8 9</Text>
+  </>`}
+/>
+
+You may apply CSS classes from the examples below to any HTML elements, according to your semantic needs.
```
Comment (AgtLucas) [inline packages/docs/src/pages/typography.mdx]: We shall remove this, as we don't rely on CSS classes over here.

File: packages/docs/src/pages/typography.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+#### Astro has a purposeful set of typographic styles that provide content organization, hierarchy patterns and consistent information architecture which is essential for good understanding and a solid user experience.
+
+## poppins
+
+Poppins is our primary font, used in the whole product’s interfaces for interactive components, contextual titles and texts. Poppins provides good legibility in small and large sizes.
+
+<Playground 
+  scope={{ Text }}
+  code={
+  `<>
```
Comment (ericaprieto) [inline packages/docs/src/pages/typography.mdx]: React fragments are automatically added by the playground component, we don't this these here

File: packages/docs/src/components/Sidebar.js (hunk 1 above)
Comment (ericaprieto) [reply packages/docs/src/components/Sidebar.js]: or we could destructuring into `function Sidebar({ pages, mdx }) {` and make it more explicit what this component expects

File: packages/docs/src/pages/typography.mdx (hunk 2 above)
Comment (tobiasbu) [reply packages/docs/src/pages/typography.mdx]: Sure. I just copy and paste from original Astro.

**Pull request merged**